source/SIL.AppBuilder.Portal.Frontend/src/ui/components/project-table/data.tsx

```diff
@@ -71,6 +71,6 @@ export function withData(WrappedComponent) {
   }
 
   return compose(
-    query(mapNetworkToProps),
+    query(mapNetworkToProps, { passthroughError: true }),
```

Comment:
later on, we may desire to make passthroughError the default, and instead add an option saying `captureError: true` for today's behavior